src/coreclr/src/vm/prestub.cpp

```diff
@@ -1917,8 +1917,8 @@ extern "C" PCODE STDCALL PreStubWorker(TransitionBlock* pTransitionBlock, Method
             {
                 pDispatchingMT = curobj->GetMethodTable();
 
-#ifdef FEATURE_ICASTABLE
-                if (pDispatchingMT->IsICastable())
+
+                if (pDispatchingMT->IsICastable() || pDispatchingMT->IsICastableObject())
```

Comment:
`pDispatchingMT->IsICastable()` is escaping the `FEATURE_ICASTABLE` condition. Unsure if it is matters, but this will break the feature build.

Reply:
>this will break the feature build

`MethodTable::IsICastable` will always return false if `FEATURE_ICASTABLE` is not defined. Or is that not what you were referring to?

Reply:
Oops. I saw the set was conditional and assumed the get was as well. Nevermind.